Guides/Python/Django-notes.md

```diff
@@ -25,6 +25,70 @@ To use a database, you should choose an Add-on from [the Data Storage category][
 ## Email
 You can't use a local SMTP server, instead choose one of our [email Add-ons][messaging-addons].
 
+## Serving Static Files
+
+Since the filesystem on cloudControl is non-persistent. We recommend to use
```

Comment:
This has nothing to do with the filesystem. The reason why you need dj-static is, that there is no webserver like Apache or Nginx inside the container to serve the static files that are part of the repository.